config/solr_wrapper_test.yml

```diff
@@ -11,6 +11,7 @@ collection:
     dir: solr/config
     name: hydra-test
 version: 6.3.0
+# validate: false # until solr_wrapper stops hardwiring the checksum URL
```

Comment:
this is commented out, you still want to put it in there commented out?

Reply:
It should be commented out, yes, but the comment is misleading. Let me amend the commit using my new git --amend superpowers.

Reply:
Updated and off to lunch. BRB